### DISCUSSION TITLE
Paraview Error

### DISCUSSION THREAD

**Discussion by nourriahi** [idaholab/moose#24644] [Q&A General] [answered]
Hello everyone, 

I am facing a problem with Paraview! I am trying to open moose simulation which is running correctly but I keep having this error message each time :

![image](https://github.com/idaholab/moose/assets/131006365/5b2ca22a-38dc-4931-a65f-244e9d476907)

Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

THis does not look right indeed. 
If you run the mesh only with `--mesh-only` in the command line, and try to open the `mesh_name_in.e` file, does it work?

Guillaume
